test/e2e/apimachinery/crd_conversion_webhook.go

```diff
@@ -122,80 +124,88 @@ var _ = SIGDescribe("CustomResourceConversionWebhook", func() {
 
 	var client clientset.Interface
 	var namespaceName string
+	var servicePort int32
 
 	ginkgo.BeforeEach(func() {
 		client = f.ClientSet
 		namespaceName = f.Namespace.Name
+		servicePort = defaultServicePort
 
 		// Make sure the relevant provider supports conversion webhook
 		framework.SkipUnlessServerVersionGTE(serverCRDConversionWebhookVersion, f.ClientSet.Discovery())
 
 		ginkgo.By("Setting up server cert")
 		context = setupServerCert(f.Namespace.Name, serviceCRDName)
 		createAuthReaderRoleBindingForCRDConversion(f, f.Namespace.Name)
-
-		deployCustomResourceWebhookAndService(f, imageutils.GetE2EImage(imageutils.Agnhost), context)
 	})
 
 	ginkgo.AfterEach(func() {
 		cleanCRDWebhookTest(client, namespaceName)
 	})
 
-	ginkgo.It("Should be able to convert from CR v1 to CR v2", func() {
-		testcrd, err := crd.CreateMultiVersionTestCRD(f, "stable.example.com", func(crd *apiextensionsv1.CustomResourceDefinition) {
-			crd.Spec.Versions = apiVersions
-			crd.Spec.Conversion = &apiextensionsv1.CustomResourceConversion{
-				Strategy: apiextensionsv1.WebhookConverter,
-				Webhook: &apiextensionsv1.WebhookConversion{
-					ClientConfig: &apiextensionsv1.WebhookClientConfig{
-						CABundle: context.signingCert,
-						Service: &apiextensionsv1.ServiceReference{
-							Namespace: f.Namespace.Name,
-							Name:      serviceCRDName,
-							Path:      pointer.StringPtr("/crdconvert"),
-							Port:      pointer.Int32Ptr(serviceCRDPort),
-						},
-					},
-					ConversionReviewVersions: []string{"v1", "v1beta1"},
-				},
+	ginkgo.Context("Running a conversion webhook service on the default port", func() {
+		ginkgo.BeforeEach(func() {
+			deployCustomResourceWebhookAndService(f, imageutils.GetE2EImage(imageutils.Agnhost), context, servicePort)
```

Comment:
would `servicePort = defaultServicePort` make more sense here instead of the top-level BeforeEach?

Reply:
latest PR update removes this part too